### DISCUSSION TITLE
Is there possibility to run this addon in docker container?

### DISCUSSION THREAD

**Discussion by lbroker** [siku2/hass-komfovent-c5#36] [General]
Is there possibility to run this addon in docker container?

**Comment by siku2** [comment]
Can you please elaborate on what you want to achieve?

**Comment by lbroker** [comment]
I use HomeAssistant installed in docker container using Synology NAS hardware. There is used image: homeassistant/home-assistant for that installation. There are also other components like deconz in separate docker.
And I want to use hass-komfovent-c5 addon, and think then how to install it in HA in this docker installation model.

**Reply by siku2** [reply]
You don't need to run custom integrations in a separate docker container. Integrations run inside of Home Assistant itself.
Please take a look at the documentation for how to install HACS: https://hacs.xyz/docs/setup/download
Once you have it installed (in your Home Assistant container) you can follow the steps outlined in the README.
